include/grpc/support/alloc.h

```diff
@@ -61,6 +61,9 @@ GPRAPI void gpr_set_allocation_functions(gpr_allocation_functions functions);
 /** Return the family of allocation functions currently in effect. */
 GPRAPI gpr_allocation_functions gpr_get_allocation_functions(void);
 
+/** Equals the minimum memory alignment required by ABI from allocators. *.
+#define GRPC_MINIMUM_ALIGNMENT (sizeof(uintptr_t) * 2)
```

Comment:
This is only helpful in the case where the first thing you want in the allocation is smaller than this.  I'd really prefer a more general-purpose macro that rounds up any arbitrary size to the next alignment size, which we can use everywhere.  I think we already have something like this in a few places.  For example:

https://github.com/grpc/grpc/blob/master/src/core/lib/surface/call.cc#L275

Reply:
Right, but that's not what the comment you were making would really use it for. Or in other words, if I use this macro, I would write `(1)`.

In fact, what I need is really akin to `GPR_MAX_ALIGNMENT` used by this function, and that's what I should do.

Reply:
I'm not sure what you mean by "if I use this macro, I would write `(1)`".

What I'm thinking here is that we should have a uniform way of handling any case where we use a singla allocation for more than one object.  So, generally speaking, whenever we are combining the allocations of two objects, A and B, we need to a way to pad the size of A up to the next alignment boundary.  In the case of this PR, object A happens to be a `size_t`, but we can treat that the same as any other object.

So, using the `GPR_ROUND_UP_TO_ALIGNMENT_SIZE()` macro from #15891, I think this PR could use something like `GPR_ROUND_UP_TO_ALIGNMENT_SIZE(size_t) + size`.